lua/mapvote/client/vgui/mapvote_panel.lua

```diff
@@ -182,6 +182,14 @@ function PANEL:CreateVoterPanel( identifier )
     iconContainer:SetMouseInputEnabled( false )
     icon:SetAlpha( 200 )
 
+    if voteMult > 1 then
+        local label = vgui.Create( "DLabel", icon )
+        label:SetText( voteMult .. "x" )
+        label:SetPos( 2, 2 )
+        label:SetColor( color_white )
```

Comment:
having the color configurable would be nice https://github.com/CFC-Servers/map_vote?tab=readme-ov-file#styling

Reply:
![gmod_r4kt5NNYDa](https://github.com/user-attachments/assets/af3b10db-37e2-49a8-afb3-4bba130375d2)
MapVote.style.colorTextPrimary for the text color and MapVote.style.colorPrimaryBG for the outline color